### ISSUE TITLE
Place HTML code inside of <body>

### ISSUE THREAD

**Issue by griffinlacek** [daksh0130-cc/prj-rev-bwfs-dasmoto#1]
It looks like the code you have didn't make it to Github or you didn't have a chance to finish the project, but what code there is should be inside the body tags! If you weren't able to finish or a Github issue occurred that it's not all there, try and finish up and I can comment more!

https://github.com/daksh0130-cc/prj-rev-bwfs-dasmoto/blob/master/Dasmoto's%20Arts%20%26%20Crafts/index.html#L5-L6
https://github.com/daksh0130-cc/prj-rev-bwfs-dasmoto/blob/master/Dasmoto's%20Arts%20%26%20Crafts/index.html#L13-L14

**Comment by daksh0130-cc** [comment]
Yeah i ran out of time but yeah I try to finish but i was really confused how to do it i try taking help from code academy advisors but still it was hard to understand
